pyvlx/api/get_state.py

```diff
@@ -22,11 +22,11 @@ def __init__(self, pyvlx: "PyVLX"):
 
     async def handle_frame(self, frame: FrameBase) -> bool:
         """Handle incoming API frame, return True if this was the expected frame."""
-        if not isinstance(frame, FrameGetStateConfirmation):
```

Comment:
Why did you flip the `if` ? I think we should keep the "return early "pattern.

Reply:
https://medium.com/swlh/return-early-pattern-3d18a41bba8 

I think we should keep the "return early" pattern here. Esp. if you get used to this type of coding, it is much easier to read. We introduced this pattern some years ago in our company and made good experiences with it.

Sorry for beeing rabulistic here :-/

Reply:
No problem, fine for me.